lib/esyBuildRuntime.sh

```diff
@@ -10,37 +14,45 @@ FG_RESET='\033[0m'
 ESY__SANDBOX_COMMAND=""
 
 if [[ "$esy__platform" == "darwin" ]]; then
-  ESY__SANDBOX_COMMAND="sandbox-exec -f $cur__target_dir/_esy_sandbox.sb"
+  ESY__SANDBOX_COMMAND="sandbox-exec -f $cur__target_dir/_esy/sandbox.sb"
 fi
 
-esy-prepare-install-tree () {
-  mkdir -p $cur__target_dir
-  mkdir -p          \
-    $cur__install   \
-    $cur__lib       \
-    $cur__bin       \
-    $cur__sbin      \
-    $cur__man       \
-    $cur__doc       \
-    $cur__share     \
+_esy-prepare-build-env () {
+
+  rm -rf $cur__install
```

Comment:
We didn't do it before. I believe this is more correct especially for builds of a root package where we put it into `$esy__sandbox/_insttmp`(notice no hash in path) — that way a previously failed builds could leave some stale artefacts and influence the new build.